youtrack_sdk/exceptions.py

```diff
@@ -13,9 +13,15 @@ class YouTrackUnauthorized(YouTrackException):
     pass
 
 
-class StrictIntError(PydanticValueError):
-    msg_template = "value is not a valid integer"
+class StrictIntError(PydanticUserError):
```

Comment:
See above, this is not good.

Reply:
We agreed to remove this in the meeting, because Pydantic V2 validatiors should raise `ValueError`: https://docs.pydantic.dev/latest/usage/validators/#field-validators.